utils.js

```diff
@@ -217,20 +217,42 @@ export function getLicenses(pkg, format = "xml") {
             licenseContent.id = l;
             licenseContent.url = "https://opensource.org/licenses/" + l;
           } else if (l.startsWith("http")) {
-            if (!l.includes("opensource.org")) {
-              licenseContent.name = "CUSTOM";
-            } else {
+            if (
+              l.includes("mit-license") ||
+              l.includes("://github.com/dotnet/standard/") ||
+              l.includes("://github.com/dotnet/corefx/") ||
+              l.includes("://github.com/dotnet/core-setup/") ||
+              l.endsWith("?linkid=869050") // https://github.com/mono/mono/blob/main/LICENSE
```

Comment:
The linked mono license reads like a custom license and not an mit license.

Reply:
Hmm, after another look I agree. I was mislead with the initial sentence. But there are even code files with different licenses linked into Mono. Will remove Mono completely.